polars/polars-lazy/Cargo.toml

```diff
@@ -132,6 +132,21 @@ binary_encoding = ["polars-plan/binary_encoding"]
 # no guarantees whatsoever
 private = ["polars-plan/private"]
 
+# all opt-in datatypes
```

Comment:
I don't think we need this on `polars-lazy`. We already have this on `polars` which will activate it on `polars-lazy`.

Reply:
Reverted with https://github.com/pola-rs/polars/pull/7290/commits/64e4ddd467e7a5cd8d6a7e5246b4ea9c88770103